stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/querychannels/DefaultChatEventHandler.kt

```diff
@@ -13,26 +13,36 @@ import kotlinx.coroutines.flow.StateFlow
 
 /**
  * Default implementation of [ChatEventHandler] which covers the default filter of channels.
- * It  adds or removes channels accordingly with the notification event received. Events handled are: [NotificationAddedToChannelEvent],
- * [NotificationMessageNewEvent], [NotificationRemovedFromChannelEvent].
- * This Handler will skip the even if the channel is already added or absent and no interaction is needed.
+ * The channel will be added if the current user is a member and it will be removed otherwise.
  *
- * This handler expects that the list of channels are the channels that we user is a member.
+ * @param channels The list of visible channels.
  */
 public class DefaultChatEventHandler(private val channels: StateFlow<List<Channel>>) :
     BaseChatEventHandler() {
 
-    /** Handles [NotificationAddedToChannelEvent] event. It adds the channel, if it is absent. */
+    /**
+     *  Handles [NotificationAddedToChannelEvent] event. It adds the channel if it is absent.
+     *
+     * @param event Instance of [NotificationAddedToChannelEvent] that is being handled.
+     * @param filter [FilterObject] which is used to define an outcome.
```

Comment:
Would be nice to add `@return` info too.

Reply:
I'll add this later